cmd/deploy/command_configuration_test.go

```diff
@@ -168,3 +169,52 @@ func Test_mergeServicesToDeployFromOptionsAndManifest(t *testing.T) {
 		})
 	}
 }
+
+func Test_switchSSHRepoToHTTPS(t *testing.T) {
+	tests := []struct {
+		name        string
+		repo        string
+		expected    *url.URL
+		expectedErr error
+	}{
+		{
+			name: "input-ssh",
+			repo: "git@github.com:okteto/go-getting-started.git",
+			expected: &url.URL{
+				Scheme: "https",
+				Host:   "github.com",
+				Path:   "okteto/go-getting-started.git",
+			},
+		},
+		{
+			name: "input-https",
+			repo: "https://github.com/okteto/go-getting-started.git",
+			expected: &url.URL{
+				Scheme: "https",
+				Host:   "github.com",
+				Path:   "/okteto/go-getting-started.git",
+			}},
+		{
+			name: "input-http",
+			repo: "http://github.com/okteto/go-getting-started.git",
+			expected: &url.URL{
+				Scheme: "https",
+				Host:   "github.com",
+				Path:   "/okteto/go-getting-started.git",
+			}},
+		{
+			name:        "input-not-allowed",
+			repo:        "github.com/okteto/go-getting-started.git",
+			expectedErr: errUnsupportedScheme,
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			url, err := switchSSHRepoToHTTPS(tt.repo)
```

Comment:
NIT: Maybe we should modify the function name to `transformURLToHTTPS` or something like that. `switchSSHRepoToHTTPS` is not the best name because we can use more schemes than `SSH`. WDYT?

Reply:
makes sense, i renamed to `switchRepoSchemaToHTTPS`